### ISSUE TITLE
Custom font not loading in react direflow application

### ISSUE THREAD

**Issue by darshna09** [Silind-Software/direflow#193]
I have created a javascript direflow application. I am trying to load a local font but haven't been able to. Below are the details and code snippets I have applied.

![image](https://user-images.githubusercontent.com/41057192/102983233-c540bb80-4531-11eb-9ece-7d0c8e854041.png)

`font.css`
```
@font-face {
    font-family: 'MyFont';
    src: local('MyFont'),
         url('./fonts/MyLocalFont.eot') format('eot'),
         url('./fonts/MyLocalFont.woff') format('woff'),
         url('./fonts/MyLocalFont.woff') format('woff');
}
```

`direflow-components/testing-component/index.js`

```
plugins: [
    {
      name: 'font-loader',
      options: {
        custom: {
          families: ['MyLocalFont'],
          urls: ['/fonts.css']
        },
      },
    },
  ],
```

`App.css`
```
.header-title {
  font-size: 34px;
  color: #5781C2;
  font-family: 'MyLocalFont';
}
```
The font files doesn't load. Please help.

**Comment by darshna09** [comment]
It is working.

Removed `font-loader` plugin from `direflow-components/testing-component/index.js` and removed quotes from the `font-family` property that is the font name.

 Changed `font-family: 'MyLocalFont';` to `font-family: MyLocalFont;`

**Comment by kelvinwongg** [comment]
@darshna09 I ran into the same problem not loading custom fonts, do you mean remove 'font-loader' import from the top of the index.js file?

**Comment by kelvinwongg** [comment]
@darshna09 Yes, I do run into the 404 font file problem, the production build point my font file and add some numbers to the filename, like so:

static/media/[Orignal Filename].08e26c81.woff

Do I have to update the filename for every build?

BTW, I found this Direflow things is a bit waste of time, because it is very **buggy and hacky**

**Comment by darshna09** [comment]
> I would rather build Web Component from scratch or using tools like Google Lit, what do you think?

@kelvinwongg I have worked on all these LitElement, Direflow - to tell the truth, there is some hassle with everything. 

For the font issue, we ended up deploying the font files on the server and then using that URL in the `@font-face`. Note the `@font-face` where then defined outside the context of Direflow. Not sure if it will work inside or not. 

We can create a bug and see what other people suggest. Do send me the link in case you open one.
